src/preferences/dialog/dlgprefeq.cpp

```diff
@@ -231,10 +231,10 @@ void DlgPrefEQ::slotPopulateDeckEffectSelectors() {
         }
         //: Displayed when no effect is selected
         box->addItem(tr("None"), QVariant(QString("")));
-        if (selectedEffectId.isNull()) {
+        if (selectedEffectId.isNull() || QString::compare(selectedEffectName, tr("None")) == 0) {
```

Comment:
Please prefer to use operator `==` if you don't need the ordering capabilities of `compare()`